uint8 gait_type
float32 velocity
float32 nominal_height
float32 stance_duration
float32 stance_depth
float32 swing_height
uint8 movement
